docs/en/latest/FAQ.md

```diff
@@ -595,6 +595,6 @@ The differences between the two are described in the table below:
 
 You can find more answers on:
 
-- [Apache APISIX Slack Channel](/docs/general/community#joining-the-slack-channel)
-- [Ask questions on APISIX mailing list](/docs/general/community#joining-the-mailing-list)
+- [Apache APISIX Slack Channel](https://apisix.apache.org/docs/general/join/#join-the-slack-channel)
+- [Ask questions on APISIX mailing list](https://apisix.apache.org/docs/general/join/#subscribe-to-the-mailing-list)
```

Comment:
```suggestion
- [Apache APISIX Slack Channel](/docs/general/join/#join-the-slack-channel)
- [Ask questions on APISIX mailing list](/docs/general/join/#subscribe-to-the-mailing-list)
```

Reply:
https://github.com/apache/apisix-website/blob/master/website/docusaurus.config.js#L307

The link is broken because of a redirect. Use the correct document path to access it normally.